<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managerWindow</class>
 <widget class="QDialog" name="managerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 77, 56);</string>
  </property>
  <widget class="QPushButton" name="addDeadBodyButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 77, 56);</string>
   </property>
   <property name="text">
    <string>Add dead body</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewDeadBodiesButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 77, 56);</string>
   </property>
   <property name="text">
    <string>View Dead Bodies</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>521</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0,0,0,0);
color: rgb(0, 77, 56);
font: 24pt &quot;Segoe MDL2 Assets&quot;;</string>
   </property>
   <property name="text">
    <string>Manger Management Panel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchDeadBodyButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 77, 56);
color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Search Dead Body</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eidtDeadBodyButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>380</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 77, 56);
color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Edit Dead Body</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteDeadBodyButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>430</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 77, 56);</string>
   </property>
   <property name="text">
    <string>Delete dead body</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LogoutBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>550</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(176, 0, 0);
color: rgb(255, 255, 255);
font: 600 14pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewVisitorButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>480</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 77, 56);</string>
   </property>
   <property name="text">
    <string>View Visitors</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addGraveButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 77, 56);</string>
   </property>
   <property name="text">
    <string>Add Grave</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>viewDeadBodiesButton</zorder>
  <zorder>addDeadBodyButton</zorder>
  <zorder>label_2</zorder>
  <zorder>searchDeadBodyButton</zorder>
  <zorder>eidtDeadBodyButton</zorder>
  <zorder>deleteDeadBodyButton</zorder>
  <zorder>LogoutBtn</zorder>
  <zorder>viewVisitorButton</zorder>
  <zorder>addGraveButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
